libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateEngine.cs

```diff
@@ -25,8 +25,8 @@ public TemplateEngine()
         /// Delegate for resolving resource id of imported lg file.
         /// </summary>
         /// <param name="resourceId">Resource id to resolve.</param>
-        /// <returns>Resolved resource id.</returns>
-        public delegate string ImportResolverDelegate(string resourceId);
+        /// <returns>Resolved resource content and absolute file path id.</returns>
+        public delegate (string, string) ImportResolverDelegate(string resourceId);
```

Comment:
Recommended ValueTuple, use `(string content , string absoluteFilePath)` instead of `(string, string)`

Reply:
resolved